go/enclave/vkhandler/vk_handler.go

```diff
@@ -2,14 +2,14 @@ package vkhandler
 
 import (
 	"crypto/rand"
+	"encoding/hex"
 	"fmt"
 
 	"github.com/ethereum/go-ethereum/accounts"
+	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/crypto/ecies"
 	"github.com/obscuronet/go-obscuro/go/common/viewingkey"
-
-	gethcommon "github.com/ethereum/go-ethereum/common"
 )
 
```

Comment:
as a general comment.
The vk_handler needs a comment describing its responsibillities.

Reply:
Added more comments